src/hipscat/io/write_metadata.py

```diff
@@ -0,0 +1,112 @@
+"""Utility functions for writing metadata files"""
+
+import json
+import os
+from datetime import datetime
+
+import numpy as np
+import pandas as pd
+import pkg_resources
+
+
+class NumpyEncoder(json.JSONEncoder):
+    """Special json encoder for numpy integer types"""
+
+    def default(self, o):
+        int_object = o
+        if isinstance(int_object, (np.int64, np.ulonglong)):
+            return int(int_object)
+
+
+def write_json_file(metadata_dictionary, file_name):
+    """Convert metadata_dictionary to a json string and print to file.
+    Args:
+        metadata_dictionary (:obj:`dictionary`): a dictionary of key-value pairs
+        file_name (str): destination for the json file
+    """
+    dumped_metadata = json.dumps(metadata_dictionary, indent=4, cls=NumpyEncoder)
+    with open(
+        file_name,
+        "w",
+        encoding="utf-8",
+    ) as metadata_file:
```

Comment:
May I rant on black? Logically, this is really should be just one line, but the choice of formatter makes it all less readable and verbose.

Reply:
I removed the trailing comma, and black now does the right thing. I think this was leftover from some copy/paste.